index.html

```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
                   </ul>
                 </li>
                 <li class="frameworks">
-                  <p class="">Frameworks</p>
+                  <h2> Frameworks </h2>
                 </li>
                 <li class="skills">
                   <h2> Skills </h2>
                 </li>
               </ul>
             </div>
-    </section>
-
-
+          </section>
 
+          <section class="contact__me__section">
+         <div class="contact__me">
+           <h3 class="contact__me__title">Contact Me</h3>
+           <p class="contact__me__text">
+               If you have an application you are interested in developing,
+               a feature that you need built or a project that needs coding.
+               I’d love to help with it
+           </p>
+           <form
+             class="contact__me__form"
+             action="https://formspree.io/f/mjvlknzp"
+             method="post"
+             id="form">
```

Comment:
- [ ]  Please as was  requested from the last reviewer please make sure that your design looks exactly like Figma's design 

| Your design | Figma |
| ------------- | ------- |
| ![image](https://user-images.githubusercontent.com/74744787/165345686-b76f05c1-3907-42ac-ac2e-b624368f9050.png) | ![image](https://user-images.githubusercontent.com/74744787/165345989-4666da42-aff1-4e26-9454-27c1ddd1a540.png) |
| - look at the distance between the title and the background
- look at the black line at the bottom of the form you don't have it in your design 
- look at the border of the button in the design there was no border to this button  | |